x-pack/platform/plugins/shared/ml/public/embeddables/types.ts

```diff
@@ -142,9 +143,7 @@ export interface AnomalyChartsDataLoadingApi {
 /**
  * Persisted state for the Anomaly Charts Embeddable.
  */
-export interface AnomalyChartsEmbeddableState
-  extends SerializedTitles,
-    AnomalyChartsEmbeddableOverridableState {}
+export type AnomalyChartsEmbeddableState = TypeOf<typeof anomalyChartsEmbeddableStateSchema>;
```

Comment:
Let's make sure we're replacing the concrete types with inferred types for other exported types in this file. For example, `AnomalyChartsEmbeddableOverridableState` on line 124.